operators/creation/ajax.md

```diff
@@ -0,0 +1,169 @@
+# ajax
+
+#### signature: `ajax(subscribe: function)`
```

Comment:
I think this signature may be wrong, I believe it's `urlOrRequest: string | AjaxRequest`

Reference: https://github.com/ReactiveX/rxjs/blob/master/src/internal/observable/dom/AjaxObservable.ts#L155